bitcoin/sign.go

```diff
@@ -129,6 +129,85 @@ func (w *BitcoinWallet) buildTx(amount int64, addr btc.Address, feeLevel wi.FeeL
 	return authoredTx.Tx, nil
 }
 
+func (w *BitcoinWallet) buildSpendAllTx(addr btc.Address, feeLevel wi.FeeLevel) (*wire.MsgTx, error) {
+	tx := wire.NewMsgTx(1)
+
+	height, _ := w.ws.ChainTip()
+	utxos, err := w.db.Utxos().GetAll()
```

Comment:
@amangale @cpacia Please take care as the precision work impacts data coming back from `Utxos`. @amangale do you see any concerns here?